### ISSUE TITLE
CLimbing animation

### ISSUE THREAD

**Issue by LordMeta** [tr7zw/NotEnoughAnimations#90]
The climbing animation for ladders is cool :D can you make it work on vines too?

**Comment by tr7zw** [comment]
Not really, because vines have no "side"(same with scaffolding/the nether vines). Of cause I could enable it for them too, but it looks really weird when the player just rotates in the air and hold onto nothing.

**Comment by ardissaps** [comment]
> Not really, because vines have no "side"(same with scaffolding/the nether vines). Of cause I could enable it for them too, but it looks really weird when the player just rotates in the air and hold onto nothing.

how about hug animation when climbing vine or similiar
